cluster/examples/kubernetes/ceph/cluster-external-management.yaml

```diff
@@ -0,0 +1,24 @@
+#################################################################################################################
```

Comment:
Actually could you add the external cluster yaml examples to [this doc](https://rook.io/docs/rook/master/ceph-examples.html#cluster-crd) as well?